phoenicis-repository/src/main/java/org/phoenicis/repository/types/GitRepository.java

```diff
@@ -176,7 +175,7 @@ public RepositoryDTO fetchInstallableApplications() {
     @Override
     public void onDelete() {
         try {
-            fileUtilities.remove(this.localFolder);
+            fileUtilities.remove(this.localFolder.getAbsolutePath());
```

Comment:
Do we also need to delete the lock file?

Reply:
I don't think it's required.